app/views/projects/_edit_form.html.erb

```diff
@@ -77,7 +77,7 @@
       </select>
     </div>
 
-    <% if @project.in_mediaflux? %>
+    <% if @project.in_mediaflux? && (current_user.superuser? || current_user.eligible_sysadmin?) %>
```

Comment:
@leefaisonr I think this may have unintended consequences of allow folks to edit the directory after the project has been saved to medaiflux. 
```suggestion
    <% if @project.in_mediaflux? %>
```